Stale read: add a doc about `tidb_read_staleness` (#7435)
pingcap/docs: system-variables.md

SET tidb_query_log_max_len = 20
```

### `tidb_read_staleness` <span class="version-mark">New in v5.4.0</span>

- Scope: SESSION
- Default value: `0`

> - This variable is used to set the time range of historical data that is allowed to be read in the current session. After setting the value, TiDB selects a timestamp as new as possible from the range allowed by this variable, and this timestamp affects all subsequent read operations. For example, if the value of this variable is set to `-5`, TiDB selects a timestamp as new as possible within a 5-second time range, while ensuring that TiKV has the corresponding historical version's data.

Reviewer: ```suggestion
- This variable is used to set the time range of historical data that TiDB can read in the current session. After setting the value, TiDB selects a timestamp as new as possible from the range allowed by this variable, and all subsequent read operations are performed against this timestamp. For example, if the value of this variable is set to `-5`, TiDB selects a timestamp as new as possible within a 5-second time range, while ensuring that TiKV has the corresponding historical version's data.
```

Author: 由于两条 comments 是重复内容，我暂时保留一个 comments 吧

> - This variable is used to set the time range of historical data that is allowed to be read in the current session. After setting the value, TiDB selects a timestamp as new as possible from the range allowed by this variable, and this timestamp affects all subsequent read operations. For example, if the value of this variable is set to `-5`, TiDB selects a timestamp as new as possible within a 5-second time range, while ensuring that TiKV has the corresponding historical version's data.

Reviewer: ```suggestion
- This variable is used to set the time range of historical data that TiDB can read in the current session. After setting the value, TiDB selects a timestamp as new as possible from the range allowed by this variable, and all subsequent read operations are performed against this timestamp. For example, if the value of this variable is set to `-5`, TiDB selects a timestamp as new as possible within a 5-second time range, while ensuring that TiKV has the corresponding historical version's data.
```
